modules/ROOT/pages/errors/all-errors.adoc

```diff
@@ -1,29 +1,109 @@
 :description: The Neo4j error codes for Neo4j version {neo4j-version}.
 
-
 [[neo4j-errors]]
-= List of all server error codes
+= List of all Neo4j 5 server error codes
+
+Error codes are Neo4j status codes returned by the server when the execution of a query fails.
+They always have the severity level `ERROR`.
+Errors are grouped based on the type of the error code:
+
+Client errors::
+    These errors are caused by the client and are usually related to the request itself.
+    Client errors have the prefix `Neo.ClientError`.
+
+Transient errors::
+    These errors are detected by the server and are usually related to some kind of database unavailability, such as limits reached, out-of-memory, timeouts, etc.
+    The error can be temporary and could therefore succeed if retrying the request.
+    Transient errors have the prefix `Neo.TransientError`.
+
+Database errors::
+    These errors are caused by the database and are usually related to the database state.
+    Database errors have the prefix `Neo.DatabaseError`.
 
 This page contains lists of all Neo4j errors, grouped by type.
 
+[[neo4j-client-errors]]
 == Client errors
 
 This is a complete list of all client errors Neo4j may return, and what they mean.
 
+[[Neo.ClientError.ChangeDataCapture.InvalidIdentifier]]
+===  `Neo.ClientError.ChangeDataCapture.InvalidIdentifier`
+
+This error occurs when the given change identifier is invalid.
+
+.Error details
+[cols="<1s,<4"]
+|===
+|Description
+a| Invalid change identifier.
+|GQLSTATUS code(s)
+m|52N16
+| Returned message(s)
+a|
+[options="header", cols="<4,<1"]
+!===
+! Message ! GQLSTATUS code
+! Invalid change identifier. ! 52N26
+! Given change identifier does not belong to this database ! 52N31
+! Given ChangeIdentifier describes a transaction that hasn't yet occurred ! 52N30
```

Comment:
When I look at the CDC tab in the Error spreadsheet the one listed as 52N30 here (CDC-005) is actually 52N32 (although it has an old comment from me and the same old message as CDC-001 which is 52N30). Also CDC-004 seems to be missing. 

I am not sure which source you have used for the table, so it can be either a misstake there or by you. The rest of the codes in this table look correct

Reply:
I think we are seeing different things in the spreadsheet. This is what I see for 52N16:

<img width="2055" alt="Screenshot 2024-09-13 at 12 09 25" src="https://github.com/user-attachments/assets/252387dc-0978-4aaa-92a4-ebb0a40d75b6">